PHPCI/Languages/lang.fr.php

```diff
@@ -67,19 +67,19 @@
     'delete_project' => 'Supprimer le projet',
 
     // Project Summary:
-    'no_builds_yet' => 'Aucun build pour le moment!',
-    'x_of_x_failed' => '%d parmis les derniers %d builds ont échoué.',
-    'x_of_x_failed_short' => '%d / %d ont échoué.',
-    'last_successful_build' => ' Le dernier build qui a réussi est %s.',
-    'never_built_successfully' => ' Aucun build n\'a été exécuté avec succès sur ce projet.',
-    'all_builds_passed' => 'Les derniers %d builds ont réussis.',
-    'all_builds_passed_short' => '%d / %d ont réussis.',
-    'last_failed_build' => ' Le dernier build en échec est %s.',
-    'never_failed_build' => ' Ce projet n\'a jamais eu un build en échec.',
+    'no_builds_yet' => 'Aucun build pour le moment !',
```

Comment:
I will prefer ```&nbsp;``` (No-break space) that specify the brevius word and punctation shuldn't be splitted.